README.md

```diff
@@ -1,2 +1,16 @@
 # hw01
-Repository for homework 01
+For the purpose of this assignment, my brief biography will follow a list format. 
+# Uninteresting Bio
+
+* The usual start, name. I am **Dalia**, a visiting PoliSci undergrad from Cairo University. It's alright if you mispronounce my name as *Daria*; that's the name of a 90's cartoon character whose personality bizarrely resembles mine.   
+# What brought me here?
+* To UChicago? A generous Egyptian billionaire was kind enough to send me and 8 other Egyptians on a one-year scholarship program suitably called under his name, The *Sawiris ISAL Program*. 
+* To this course in particular? My minor is social science computing yet I have only been exposed to Microsoft Access, Gephi, and Python. I was thus extremely thrilled to take this course to finally be able to conduct statistical analysis for future research. 
+
+# Interests
+My interests are pretty typical: writing, reading, drawing, walking, baking, and, only recently, [Arabic calligraphy](http://islamveateizm.com/arabic-calligraphy-riqa-minimalist/241/alluring-arabic-calligraphy-riqa-fresh-at-painting-lighting-design-pin-by-on-pinterest/).  
+***
+I don't know about you, but I'm personally wary that in the next couple of weeks, I'll be a version of this:
+
+![A Gruesome Premonition](https://media1.tenor.com/images/2a4624f017abfca893b84e5886c2d7da/tenor.gif)
```

Comment:
Yeah, that's likely to happen. But everyone's in the same boat, so you aren't alone!